app/assets/stylesheets/facets-tagging.scss

```diff
@@ -0,0 +1,15 @@
+.new_facets_tagging_update_form {
+  .facets_tagging_update_form_notification_message {
```

Comment:
I'm not sure about the naming of these classes. It's perhaps worth double-checking them with someone in the frontend community.

Reply:
These are generated class names from the simple form builder in formtastic gem. I'm just referencing them.